app/views/users/show.html.erb

```diff
@@ -17,6 +17,29 @@
     </p>
   </div>
 
+  <div class='review-block'>
+    <div class='section-head'>
+      <h3>Highlighted Reviews:</h3>
+    </div>
+    <% if @user.highlighted_reviews %>
```

Comment:
Do we want to have the reviews labled as highest and lowest? not a direction, just something for us to think about

Reply:
I like it.  Let me make teh change and repush.

Reply:
actually, let's discuss.  I've relabeled them in the test to add clarity, but I think actually labeling them outside the class as such would require two different methods which seems excessive.  Open to your ideas though.